src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/Subscriptions/DependencySubscriptionsHost.cs

```diff
@@ -274,11 +273,7 @@ private void OnDependenciesSnapshotChanged(IDependenciesSnapshot snapshot)
             DependenciesSnapshot newSnapshot;
```

Comment:
This method is not a async method but the method is called `UpdateDependenciesSnapshotAsync`